src/api/cvc4cpp.cpp

```diff
@@ -5188,6 +5187,12 @@ ExprManager* Solver::getExprManager(void) const { return d_exprMgr.get(); }
  */
 SmtEngine* Solver::getSmtEngine(void) const { return d_smtEngine.get(); }
 
+/**
+ * !!! This is only temporarily available until the parser is fully migrated to
+ * the new API. !!!
+ */
+Options& Solver::getOptions(void) { return d_smtEngine->getOptions(); }
```

Comment:
What is the parser going to use in the future?

Reply:
It's not clear. 

I think there is a fundamental strange issue that the parser is designed to be a user of the API, but the options need to be used both internally and by the parser.

If we think that options (from the SmtEngine) are needed permanently, I can update this comment.

Reply:
Resolved in offline discussion.